cmd/copilot/main.go

```diff
@@ -81,7 +81,7 @@ func buildRootCmd() *cobra.Command {
 	cmd.AddCommand(cli.BuildSvcCmd())
 	cmd.AddCommand(cli.BuildJobCmd())
 	cmd.AddCommand(cli.BuildTaskCmd())
-	cmd.AddCommand(cli.BuildLocalRunCmd())
+	cmd.AddCommand(cli.BuildRunLocalCmd())
```

Comment:
I think the help menu will show up like
```
$ copilot -h
Development
   run     Run workloads locally
```

People _might_ (or might now) be confused about its relationship with the existing `task run` and `job run`, which is probably fine.

Reply:
i think that's a fair concern; hopefully our descriptions are clear enough:
```
$ copilot task run -h
Run a one-off task on Amazon ECS.

$ copilot job run -h
Invoke a job in an environment.
```